backend/db/migrations/20210717101512-addPolicies.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.createTable('policies', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        shop_id: {
```

Comment:
We should probably add an index on shop_id since this table will likely get queried by shop_id?

Reply:
If by "add[ing] an index on shop_id", you meant to suggest adding the line `primaryKey: true`, it is done.

Reply:
You can't have 2 primary keys on a table. id is already your primary key. So you have to add an index on shop_id. something like `await queryInterface.addIndex('policies', ['shop_id'], { unique: true })`
Look into the dshop/backend/db/migrations directory, there are plenty of examples